packages/asset/src/Asset.chain.ts

```diff
@@ -78,44 +78,66 @@ export async function dispatchCreateToChain(
   }
 }
 
-export async function dispatchCreateVcToChain(
+export async function prepareCreateVcExtrinsic(
   assetQty: number,
   digest: string,
   creator: DidUri,
   authorAccount: CordKeyringPair,
   authorizationUri: AuthorizationUri,
-  assetEntryUri: AssetUri,
   signCallback: SignExtrinsicCallback
-): Promise<AssetUri> {
+): Promise<SubmittableExtrinsic> {
   try {
-    const api = ConfigService.get('api')
-    const authorizationId: AuthorizationId = uriToIdentifier(authorizationUri)
+    const api = ConfigService.get('api');
+    const authorizationId: AuthorizationId = uriToIdentifier(authorizationUri);
 
-    const tx = api.tx.asset.vcCreate(
-      assetQty,
-      digest,
-      authorizationId
-    )
+    const tx = api.tx.asset.vcCreate(assetQty, digest, authorizationId);
 
     const extrinsic = await Did.authorizeTx(
       creator,
       tx,
       signCallback,
       authorAccount.address
     )
+    return extrinsic;
+  } catch (error) {
+    const errorMessage =
+      error instanceof Error ? error.message : JSON.stringify(error);
+    throw new SDKErrors.CordDispatchError(
+      `Error preparing VC Asset Entry extrinsic: "${errorMessage}".`
+    );
+  }
+}
 
-    await Chain.signAndSubmitTx(extrinsic, authorAccount)
 
-    return assetEntryUri
+export async function dispatchCreateVcToChain(
+  assetQty: number,
+  digest: string,
+  creator: DidUri,
+  authorAccount: CordKeyringPair,
+  authorizationUri: AuthorizationUri,
+  assetEntryUri: AssetUri,
+  signCallback: SignExtrinsicCallback
+): Promise<AssetUri> {
+  try {
+    const extrinsic = await prepareCreateVcExtrinsic(
+      assetQty,
+      digest,
+      creator,
+      authorizationUri,
```

Comment:
`prepareVcExtrinsic` expects `authorAccount` as a parameter after `creator`, which is missing while calling the method.

Reply:
Done !